fre/pp/configure_script_yaml.py

```diff
@@ -87,9 +87,9 @@ def quote_rose_values(value):
     boolean, in which case do not quote them.
     """
     if isinstance(value, bool):
-        return(f"{value}")
+        return f"{value}"
```

Comment:
Since it's just returning the value  now, I think we can remove the `f"..."` and just have `value`

Reply:
That's fine too, it works and it's not a problem having f-strings at all